<!DOCTYPE HTML>
<!--
	Solid State by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<head>
    <title>{{ _('Synthetic biology software - iGem Tübingen') }}</title>
    <meta charset="utf-8"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no"/>
    <link rel="stylesheet" href="{{ url_for('static', filename='assets/css/main.css') }}"/>
    <link rel="stylesheet" href="{{ url_for('static', filename='assets/css/images.css') }}"/>
    <link rel="stylesheet" href="{{ url_for('static', filename='assets/css/magnific-popup.css') }}">
    <!-- SEO -->
    <meta name="og:title" property="og:title" content="{{ _('Synthetic biology software - iGem Tübingen') }}" />
    <meta name="og:type"property="og:type" content="website" />
    <meta property="og:url" content="https://igem-tuebingen.com/software"/>
    <!--TODO: LOGO fehlt noch -->
    <!--<meta name="og:image" property="og:image" content="https://igem-tuebingen.com/static/images/project/application.png" />-->
    <meta name="og:description" property="og:description" content="{{ _('The iGem team Tübingen subteam drylab produces synthetic biology software. Synthetic biology bioinformatics are needed.') }}" />
    <meta name="description" content="{{ _('The iGem team Tübingen subteam drylab produces synthetic biology software. Synthetic biology bioinformatics are needed.') }}" />

    <noscript>
        <link rel="stylesheet" href="{{ url_for('static', filename='assets/css/noscript.css') }}"/>
    </noscript>
</head>
<body class="is-preload">

<!-- Page Wrapper -->
<div id="page-wrapper">

    <!-- Header -->
    <header id="header">
        <h1><a href="{{ url_for('index') }}">{{ _('iGEM Team Tübingen 2019 GLP.exe') }}</a></h1>
        <nav>
            <a href="#menu">Menu</a>
        </nav>
    </header>

    <!-- Menu -->
    {% include 'menu.html' %}

    <!-- Wrapper -->
    <section id="wrapper">
        <div class="backi">
            <header>
                <div class="inner">
                    <p></p><br>
                    <p></p><br>
                    <p></p><br>
                    <p></p><br>
                    <div class="subtitle">
                        <h2>{{ _('Synthetic biology software - iGem Tübingen') }}</h2>
{#                        <p>"Life is a DNA software system." --Craig Venter</p>#}
                    </div>
                    <p></p><br>
                    <p></p><br>
                    <p></p><br>
                </div>
            </header>

            <!-- Content -->
            <div class="wrapper">
                <div class="inner">

                    <h2 class="major">{{ _('Synthetic biology software - iGem Tübingen 2018') }}</h2>
                    <ul id="image-popups">
                    <p><figure>
                            <div  class="image left">
                                <a style="font-size: small" href="{{ url_for('static', filename='images/software/BERT.png') }}" data-effect="mfp-zoom-out" title="BERT workflow">
                                    <img src="{{ url_for('static', filename='images/software/BERT.png') }}"/>
                                    Tap for more details
                                </a><figcaption style="font-size: small">{{ _('Overview of BERT´s components and workflow.') }}</figcaption>
                            </div>
                    </figure>
                        {{ _('The software project of the iGEM Team Tübingen 2018
                        addressed the problem of biopharmaceuticals being targeted by the human immune system. Upon
                        contact with foreign proteins, the immune system can trigger the production of anti-drug antibodies
                        (ADAs) by plasma cells, which neutralize the
                        function and reduce the bioavailability of biopharmaceuticals in the system.
                        We constructed the powerful and modular deimmunization workflow “Boosted Epitope
                        Reduction based on Trees” (BERT), automating protein deimmunization to solve this problem.
                        Hover over the image on the left for an overview of BERT.
                        BERT computes the immunity based on MHC class II epitope using NetMHCIIpan and minimizes the
                        immunity by epitope destroying amino acid substitution.
                        To ensure the structural integrity of the protein, highly conserved amino acids
                        were excluded from the substitution process.
                        Furthermore, BERT computes the Gibbs free energy change (ddG) for each amino acid substitution
                        and chooses the final amino acid substitutions based on the minimum ddG and minimum immunity of the
                        protein.
                        Moreover, we created a fast and highly accurate MHC class I prediction tool called MHCBoost,
                        which achieves accuracy benchmarks in the range of state of the art MHC class I prediction tools.
                        MHCBoost uses gradient boosted trees to predict short MHC binding peptides that have the
                        potential to trigger an immune response with unparalleled speed.') }}</p>

                    <p>{{ _('Links to GitHub repositories:') }}<br>
                        <a href="https://github.com/igemsoftware2018/Team_Tuebingen_BERT">{{ _('View BERT on GitHub') }}</a><br>
                        <a href="https://github.com/igemsoftware2018/Team_Tuebingen_MHCBoost">{{ _('View MHCBoost on GitHub') }}</a><br>
                    </p>


                    <h2 class="major">{{ _('Synthetic biology software - iGem Tübingen 2019') }}</h2>
                    <p>{{ _('The CRISPR/Cas system is a very powerful tool for genome editing in biological research and in treatment of diseases.
                        Besides its high specificity, off-target activity often yields unwanted side effects.
                        Therefore, the identification of off-target sites is a crucial point in many experimental set-ups.
                        To facilitate the analysis of experimental off-target identification, the iGEM Team Tübingen 2019 creates a data analysis pipeline for data generated in GUIDE-Seq experiments,
                        which is an unbiased and genome-wide method to identify off-target sites in CRISPR/Cas experiments.
                        Our goal is to generate a reproducible and parallelizable analysis software tool, based on the bioinformatics workflow management system Nextflow.
                        Nextflow allows the development of portable and reproducible workflows which can be deployed on various platforms for execution.
                        A key property of this workflow management system is its ability to decouple workflow components by isolating their dependencies with containers.
                        This ensures that no future requirement conflicts occur between different components, which allows maintaining the reproducibility of results.
                        To us, creating an easy-to-use software tool is of highest priority.
                        We want to achieve a good and intuitive usability without the user being required to do any additional programming or setup.
                        For a considerable number of useful bioinformatic tools installation and handling is not quite intuitive.
                        This potentially limits the number of users and therefore decreases the impact of the tool.
                        Therefore, we are creating a clear and comprehensible graphical user interface.
                        We also plan to improve the existing workflow layout, by replacing single components to improve the accuracy of the result and to reduce the computational time needed.
                        Another main goal is to validate the results of our workflow experimentally and to integrate the results of our software tool into our wet-lab experiments.
                        We also want to encourage other iGEM teams and research groups, who are working with CRISPR/Cas systems and are concerned about potential off-target sites, to use our tool.') }}
                    </p></ul>

                    <section class="features">
                        {% include 'banner/about.html' %}
                        {% include 'banner/project.html' %}
                    </section>

                </div>
            </div>
        </div>
    </section>

    <!-- Footer -->
    {% include 'footer.html' %}

</div>

<!-- Scripts -->
<script src="{{ url_for('static', filename='assets/js/jquery.min.js') }}"></script>
<script src="{{ url_for('static', filename='assets/js/jquery.scrollex.min.js') }}"></script>
<script src="{{ url_for('static', filename='assets/js/browser.min.js') }}"></script>
<script src="{{ url_for('static', filename='assets/js/breakpoints.min.js') }}"></script>
<script src="{{ url_for('static', filename='assets/js/util.js') }}"></script>
<script src="{{ url_for('static', filename='assets/js/spam.js') }}"></script>
<script src="{{ url_for('static', filename='assets/js/main.js') }}"></script>
<script src="{{ url_for('static', filename='assets/js/jquery.magnific-popup.js') }}"></script>
<script src="{{ url_for('static', filename='assets/js/popup_images.js') }}"></script>
<script src="{{ url_for('static', filename='assets/js/fluidImages.js') }}"></script>

</body>
</html>
